net/unbound/files/unbound-odhcpd.defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ "$(uci get dhcp.odhcpd.maindhcp)" = "1" ] || exit 0
```

Comment:
This needs to be sure the installed package is not `odhcpd+ipv6_only`. It must be `odhcpd` plain. What if dnsmasq is still enabled? Unbound and dnsmasq will contend for privileged port 53 and not everyone will want the same solution. The package README.md provides two rather easy options, so those at least must be considered.

Reply:
I understand that the way this package was built is to provide flexibility to power-users. However, in this way it is unsuitable for inclusion by default in images eg. generated on `chef.libremesh.org`. It's just be great if things would work just by installing packages without having to manually intervene and configure stuff (I know this can be done by including stuff via `FILES=...` and in a lot of ways, I still consider it a very high barrier compared to selecting a package in a browser-UI)

Another option would be to have dedicated packages only containing UCI defaults, so users may choose `unbound-dhcp-odhcpd` (or `unbound-dhcp-dnsmasq`, in case anyone really wants that).

Reply:
Buy you can NOT have that. OpenWrt default is to have dnsmasq installed using port#53. Unbound wants to use port#53. So which one wins? How do you know that this user wants overall? Maybe they just disable dnsmasq and use Unbound as is? Maybe the serialize the two? Maybe the use odhcpd? Maybe DHCP and static resources are handled by NSD with proper authoritative definition?